PowerEditor/src/ScintillaComponent/ScintillaEditView.h

```diff
@@ -170,7 +171,7 @@ const std::vector<std::vector<const char*>> g_nonPrintingChars =
 int getNbDigits(int aNum, int base);
 //HMODULE loadSciLexerDll();
 
-TCHAR * int2str(TCHAR *str, int strLen, int number, int base, int nbChiffre, bool isZeroLeading);
+TCHAR * int2str(TCHAR *str, int strLen, int number, int base, int nbDigits, bool isZeroLeading, bool isSpaceLeading);

```

Comment:
Use the enum instead of 2 boolean for keeping the consistence.

Reply:
OK, enum causes additional complication, the "worst" part of that was adding a `#include` in a file that previously didn't need it.  We'll see how you like my change; if not liked then I will ask for additional clarification on what would be liked.